### DISCUSSION TITLE
How to create a single database multi-tenant application using Lunar PHP package?

### DISCUSSION THREAD

**Discussion by misbahansori** [lunarphp/lunar#1821] [Q&A]
Hi, I want to create a single database multi-tenant application using the Lunar PHP package. But I'm not sure how to do it. Usually, each table has a `tenant_id` column to differentiate the data between tenants. How can I achieve this using the Lunar PHP package? 
Thanks in advance.

**Comment by chrillep** [comment]
@glennjacobs there are no references in the lunar docs about this.
Is it as "simple" as to follow the filament v3 documentation about tenancy.

refs
* https://filamentphp.com/docs/3.x/panels/tenancy
* https://redberry.international/lunarphp-headless-ecommerce/

**Comment by glennjacobs** [comment]
A single database multi-tenant application with Lunar isn't currently possible.

You would have to use a landlord/tenant DB set-up, see https://tenancyforlaravel.com/

It's something we're considering for a future version.

**Reply by chrillep** [reply]
@glennjacobs aaah great thanks for the direction. ❤️
